src/modules/0.0.18/aws_cloudfront/entity/distribution.ts

```diff
@@ -0,0 +1,90 @@
+import {
+    Column,
+    Entity,
+    PrimaryGeneratedColumn,
+} from 'typeorm'
+
+import { cloudId, } from '../../../../services/cloud-id' // This is ridiculous. Can we fix this?
+
+export enum viewerProtocolPolicyEnum {
+    ALLOW_ALL = "allow-all",
+    REDIRECT_TO_HTTPS = "redirect-to-https",
+    HTTPS_ONLY = "https-only"
+}
+
+export enum originProtocolPolicyEnum {
+    HTTP_ONLY = "http-only",
+    MATCH_VIEWER = "match-viewer",
+    HTTPS_ONLY = "https-only"
+}
+
+@Entity()
+export class Distribution {
```

Comment:
I had never used Cloudfront, but this data model is more complex than I anticipated. Not related to this PR, but it might also make sense to have a high level form of this module in the future

Reply:
we were discussing about dividing between distribution and distribution_metadata, but as a follow-up